recipes.js

```diff
@@ -2,10 +2,126 @@ const mongoose = require('mongoose');
 const Schema   = mongoose.Schema;
 const data = require('./data.js')
```

Comment:
Here you imported everything from data.js and saved it in variable data; this is what you'll use for your insertMany() method.